package.json

```diff
@@ -1,10 +1,14 @@
 {
   "scripts": {
+    "jsdeps": "python -I scripts/jsdeps.py",
```

Comment:
I wonder if we should get rid of this. We're not using it anymore, and solely rely on the hatch build hook to do this.

Reply:
It could be useful for syncing `third-party` folder when installing new node modules.

Even if the same result can be achieved by running directly the `Python` script, having it defined in `package.json` is straightforward for anyone looking what `npm` scripts are available. 
*(personally, I always look inside `package.json` to see what scripts can be run)*

Let's consider the simplicity: 
- `npm install xxx --save`
- `npm run jsdeps`

than:
- `npm install xxx --save`
- `python -I scripts/jsdeps.py`

Reply:
Sure ok, we can keep it.

But if we update anything in requirements.txt we also have to re-run `pip install -e .`, and updating the js deps for Fontra is more similar to that in my view.

Reply:
Yes, it's bidirectional.